DEA_datasets/DEA_Water_Observations.ipynb

```diff
@@ -0,0 +1,1409 @@
+{
```

Comment:
<p>This table doesn't show up on ReviewNB so I'm assuming it works OK in Jupyter.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/830/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>